env_backend/README.md

```diff
@@ -0,0 +1,82 @@
+# Backend Flask server for Friendymap
+
+The backend server uses the Flask framework.
+This documentation explains how to configure and test backend server.
+
+
+## Environment requirements
+
+You need to install and update the latest python version in your computer.
+
+
+To check the version installed, open a terminal window and entering the following:
+
+>**_python --version_** 
+
+**Note**: _python/python2 should be pre-installed, if not, you may download and install from [Python2 download](https://www.python.org/downloads/release/python-2718/).
+
+>**_python3 --version_**
+
+**Note**: _python3 download address [Python3 download](https://www.python.org/downloads/release/python-3111/).
+ 
+
+
+## Configuring
+
+To configure backend server, you need to:
+
+- Create an Environment
+- Activate the Environment
+- Install Flask
+
+### _Create an Environment_
+1.Make a separate directory for your project:
+
+  >**_mkdir Friendymap_**  
+
+2.Move into the directory:
+
+  >**_cd Friendymap_** 
+
+3.Within the directory, create the virtual environment for Flask. When you create the environment, a new folder appears in your project directory with the environment’s name.
+
+  >**_python3 -m venv (name of environment)_**
```

Comment:
You may decide what to call your environment, since it's a flask framework, you may name it like "flask_env" or just "env_backend", basically whatever name works for you.